lib/elixir/lib/module/checker.ex

```diff
@@ -157,6 +173,14 @@ defmodule Module.Checker do
     ]
   end
 
+  defp format_warning({:deprecated, module, function, arity, reason}) do
+    [
+      Exception.format_mfa(module, function, arity),
+      " is deprecated. ",
+      reason
```

Comment:
reason may be "true" (`@deprecated true`). We also may want to format deprecated structs better.

Reply:
https://github.com/elixir-lang/elixir/blob/emj/gp-deprecations/lib/elixir/lib/module.ex#L1956

Reply:
We should probably fix this one then: https://github.com/elixir-lang/elixir/blob/master/lib/elixir/src/elixir_dispatch.erl#L381 ?